README.md

```diff
@@ -2,31 +2,379 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Understand how Java manages memory
+- Create multiple references to one object
+- Introduce Java's garbage collector
 
-## Introduction
+## Memory Management
 
-A 1-2 sentence summary of what will be covered.
+In Java, objects live in **memory**. When we start the JVM, some memory from the operating system (OS) will be
+allocated to the JVM. Once the memory is allocated, we can use that memory to run our Java program. Memory is an
+important concept because it helps us create new objects and remove them.
 
-## Topic 1
+Think of a sandbox. We can build a castle with the sand inside the sandbox. When we are done, we can destroy the
```

Comment:
I'm happy you kept my sandbox analogy lol.

Reply:
It's a solid analogy!